pkg/controller/framework/controller.go

```diff
@@ -94,7 +98,38 @@ func (c *Controller) Run(stopCh <-chan struct{}) {
 
 	r.RunUntil(stopCh)
 
+	// Spawn a number of workers.
+	// TODO: Make it an argument.
+	c.workers = 16
+	wg := sync.WaitGroup{}
+	wg.Add(c.workers)
+	c.workQueue = make([]chan interface{}, c.workers)
+	for i := 0; i < c.workers; i++ {
+		// TODO: What should be the size of channel?
```

Comment:
I think we're pretty light memory-wise, so we can afford making this 30-60k pods total (150MB?)